src/daemon/daemon_init_settings.cpp

```diff
@@ -53,6 +53,7 @@ QString persistent_settings_filename()
 
 QString driver_interpreter(QString val)
 {
+    val = val.toLower().remove("-");
```

Comment:
This also accepts something like `HyPeR-----V`, right? Is that ok?  I think we might need a regex format check on the input val first and then interpret.

Reply:
> Is that ok?

Since hyper-v is the only hypervisor that contains dashes IIRC, it might be better to only apply toLower() to `val` and then whenever there is a check for `if (driver == "hyperv")` to also check for `if (driver == "hyperv" || driver == "hyper-v")`. What do you think about this solution instead?

Reply:
I would like to add a third option :slightly_smiling_face:  How about we do only `toLower()` here and convert hyper-v to hyperv on the Windows platform?

Reply:
That could even be done in a separate PR, to avoid having to sync branches.

Reply:
> Since hyper-v is the only hypervisor that contains dashes IIRC, it might be better to only apply toLower() to val and then whenever there is a check for if (driver == "hyperv") to also check for if (driver == "hyperv" || driver == "hyper-v"). What do you think about this solution instead?

This sounds good. One more hard-coded string is not that bad in this scenario.